server/models/db/workerTimeslot.js

```diff
@@ -0,0 +1,20 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+const Tables = require('../../constants/tables.json');
+const TimeslotStatus  = require('../../constants/timeslot-status.json');
+
+// This method inserts an appointment entry given specific information.
+async function insertWorkerTimeslot(slotId, schoolId, workerId, status, date) {
+    // Added conditional statements to prevent status and date from storing "null" instead of null. 
+    // Date is currently being input as a string. Unsure of correct format for date data type. 
+    // Opportunity to add the "recurring" functionality not taken yet. Unsure if this will come from the front end or not. 
+    //var date_format = moment(date).format('YYYY MM DD');
+    //console.log(date_format); 
+
+    return db.any(`insert into ${Tables.workerTimeslot} (slot_id, school_id, worker_id, status, date) values (${slotId}, ${schoolId}, ${workerId}, ${status ? `'${status}'` : `'available'`} , '${date}');`);
```

Comment:
Hi Peter,
Instead of hardcoding `available`, I would recommend using`TimeslotStatus.available` to leverage the constant files and prevent hardcoding.

Reply:
Amy, based on time constraints, I would suggest leaving as is for the time being and revisiting at a later point when we make changes/additions that are relevant.

Reply:
Hi Peter,
The required change for this line would be: `${status ? `'${status}'` : `'${TimeslotStatus.available}'`}`.